functions.yml

```diff
@@ -457,6 +462,9 @@ gwas_functions:
         - name: test
           doc: Which logistic regression test to use. Can be ``LRT`` or ``Firth``
           type: str
+        - name: offset
+          doc: An optional double array of offset values
```

Comment:
Can you briefly explain what the offset does?

Reply:
Added